packages/logging/src/logging-middleware.ts

```diff
@@ -85,15 +108,23 @@ export function expressRequestLoggingMiddleware(
             req.caller = caller
         }
         const childOptions = { requestId: req.requestId }
+
+        // To see if we will be overriding the base sampler.
+        const fullPath = req.baseUrl + req.path
```

Comment:
I'd just use the `req.path` since `req.baseUrl` wouldn't change for an express app